### DISCUSSION TITLE
How to choose the version of cuda when install deepmd with conda?

### DISCUSSION THREAD

**Discussion by bigd4** [deepmodeling/deepmd-kit#225] [Q&A] [answered]
Hi, the latest driver version of our gpu is 384.90 and so cuda is 9.0. And administrator is reluctant to upgrade drivers. So is there some way to choose the version of tensorflow and cuda when use
 `conda install deepmd-kit=*=*gpu lammps-dp=*=*gpu -c deepmodeling`?
Can you give us some suggestions to resolve this issue? Thanks!

**Comment by njzjz** [comment, marked as answer]
You need to compile it by yourself.
